Assets/Scripts/GUI/PlaySceneUI.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class PlaySceneUI : MonoBehaviour
+{
+    [SerializeField] private float windowResizeTime = 1f;
```

Comment:
I believe that a faster windowResizeTime feels better. 0.2f

Reply:
changed it to 0.4 in PR #27